files/rsyslog/rsyslog.sh

```diff
@@ -94,7 +94,7 @@ else
         else
             export SKIP_EMPTY=false
             export USE_MMEXTERNAL=true
-            echo "INFO: Both USE_MMEXTERNAL and SKIP_EMPTY were set to false, but the UNDEFINED fields were configured. Enabling USE_MMEXTERNAL. Consider using USE_MMEXTERNAL=true to avoid this message in the future."
+            echo "INFO: Both USE_MMEXTERNAL and SKIP_EMPTY were set to false, but the environment variables to manage undefined and empty fields were configured. Enabling USE_MMEXTERNAL. Consider using USE_MMEXTERNAL=true to avoid this message in the future."
```

Comment:
Do we need _ to avoid this message in the future_? The other messages do not have this statement. I think they make sense without that statement.

Reply:
All right.  Let me remove that part _ to avoid this message in the future_.